### ISSUE TITLE
php error on Ask a Question form

### ISSUE THREAD

**Issue by mashanadavis** [NIHL/Drupal7-Migration#34]
http://nihlibrarystg.prod.acquia-sites.com/admin/content/webform

**Comment by jronmorgan** [comment]
I believe this has been cleaned up..

BTW, I am unable to see the above link when not logged in. Is that as expected?

**Comment by mashanadavis** [comment]
I cannot get to the form as the URL now reads: http://nihlibrarystg.prod.acquia-sites.com/node/node/4

**Comment by mashanadavis** [comment]
An error still exists on the page. Removed the extra /node/ in the URL. 
![error-forms-askaquestion-unidentified-index-fields fw](https://cloud.githubusercontent.com/assets/13887931/10102058/75f9f4fa-636a-11e5-98ed-1622a3490905.png)

**Comment by tgifford-webfirst** [comment]
This is a bug in the jump menu views plugin: https://www.drupal.org/node/2272387. There is a candidate patch, which I have applied; this will need to be tracked for future updates to the views module. I added a patch.make file as specified in [the documentation](https://docs.acquia.com/ra/patching), but that doesn't guarantee this patch won't get reverted in the future by a dev that's not aware. It's also likely that a better patch will be written, in which case this would need to be replaced.
